packages/dashmate/src/docker/DockerCompose.js

```diff
@@ -16,17 +16,21 @@ const ServiceAlreadyRunningError = require('./errors/ServiceAlreadyRunningError'
 const ServiceIsNotRunningError = require('./errors/ServiceIsNotRunningError');
 const ContainerIsNotPresentError = require('./errors/ContainerIsNotPresentError');
 
-const { HOME_DIR_PATH, PACKAGE_ROOT_DIR } = require('../constants');
+const { PACKAGE_ROOT_DIR } = require('../constants');
 
 class DockerCompose {
+  #homeDir;
```

Comment:
Is there a specific reason to use private fields? I don't mind it, but it in this case it looks a bit messy because 
 it mixes up styles. F.e this.docker and this.startedContainers does not use private fields.

Reply:
Migrated the entire class to ES6 style